src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md

```diff
@@ -2,7 +2,7 @@ project_path: /web/fundamentals/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 description: If your site has a ton of images and video, but you don't want to cut down on any of it, lazy loading might be just the technique you need to improve initial page load time and lower per-page payload.
 
-{# wf_updated_on: 2018-03-28 #}
+{# wf_updated_on: 2018-05-28 #}
```

Comment:
If this is from today wouldn't it be 2018-04-04?

Reply:
Late night commits.. :) Fixed hopefully!